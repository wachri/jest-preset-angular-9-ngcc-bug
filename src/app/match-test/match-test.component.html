<p>match-test works!</p>
