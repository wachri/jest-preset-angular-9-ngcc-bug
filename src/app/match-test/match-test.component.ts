import { Component, OnInit } from '@angular/core';

@Component({
  selector: 'app-match-test',
  templateUrl: './match-test.component.html',
  styleUrls: ['./match-test.component.sass']
})
export class MatchTestComponent implements OnInit {

  constructor() { }

  ngOnInit(): void {
  }

}
