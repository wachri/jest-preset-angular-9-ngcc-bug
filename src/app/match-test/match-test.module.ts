import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { MatchTestComponent } from './match-test.component';



@NgModule({
  declarations: [MatchTestComponent],
  imports: [
    CommonModule
  ]
})
export class MatchTestModule { }
