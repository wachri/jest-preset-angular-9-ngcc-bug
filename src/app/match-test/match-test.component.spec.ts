import { async, ComponentFixture, TestBed } from "@angular/core/testing";

import { MatchTestComponent } from "./match-test.component";
import { prepareTestEnvironment } from "@hetznercloud/ngx-prepare-test-environment";

describe("MatchTestComponent", () => {
  let component: MatchTestComponent;
  let fixture: ComponentFixture<MatchTestComponent>;

  prepareTestEnvironment({
    declarations: [MatchTestComponent]
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(MatchTestComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it(`should match snapshot`, () => {
    expect(fixture).toMatchSnapshot();
  });
});
