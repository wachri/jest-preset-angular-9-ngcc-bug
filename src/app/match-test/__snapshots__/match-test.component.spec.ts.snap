// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MatchTestComponent should test against snapshot 1`] = `
<app-match-test>
  <p>
    match-test works!
  </p>
</app-match-test>
`;
